Doc/library/readline.rst

```diff
@@ -27,23 +27,26 @@ Readline library in general.
 .. note::
 
   The underlying Readline library API may be implemented by
-  the ``libedit`` library instead of GNU readline.
+  the ``editline`` (``libedit``) library instead of GNU readline.
   On macOS the :mod:`readline` module detects which library is being used
   at run time.
 
-  The configuration file for ``libedit`` is different from that
+  The configuration file for ``editline`` is different from that
   of GNU readline. If you programmatically load configuration strings
-  you can check for the text "libedit" in :const:`readline.__doc__`
-  to differentiate between GNU readline and libedit.
+  you can use :data:`backend` to determine which library is being used.
 
-  If you use *editline*/``libedit`` readline emulation on macOS, the
+  If you use ``editline``/``libedit`` readline emulation on macOS, the
   initialization file located in your home directory is named
   ``.editrc``. For example, the following content in ``~/.editrc`` will
   turn ON *vi* keybindings and TAB completion::
 
     python:bind -v
     python:bind ^I rl_complete
 
+.. data:: backend
```

Comment:
Out of curiosity, do we have to export the backend name as the public?

Reply:
Because users will have the same issue - we have a big note section to explain this in our docs.

Reply:
Ah okay @encukou commented about this: https://github.com/python/cpython/pull/107748#discussion_r1395432432